sessions/css-layout-with-flexbox/navigation/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>HTML & CSS</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+	</head>
+	<body>
+		<header>
+			<h1>My Personal Website</h1>
+			<nav>
+				<a class="button" href="#about">About</a>
```

Comment:
I'm not sure if giving an anchor tag the class "button" is advisable. 🤔

Reply:
Good catch